internal/pkg/earpointer/analysis.go

```diff
@@ -532,6 +572,41 @@ func (vis *visitor) visitBuiltin(builtin *ssa.Builtin, instr ssa.Instruction) {
 	}
 }
 
+// Handle some known functions.
+func (vis *visitor) visitKnownFunction(fn *ssa.Function, instr ssa.Instruction) bool {
```

Comment:
AFAICT, ideally this would be expanded to include all other relevant standard library functions. Can we add a TODO with an issue number to track this?